examples/jetpack_compose/WORKSPACE

```diff
@@ -1,3 +1,13 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+_COMPOSE_VERSION = "1.1.0-beta03"
```

Comment:
I don't have a strong opinion on this, but should we maybe target the last stable instead?

Reply:
It's complicated because they are tied to the Kotlin compiler. I made note of that in the description. We can pass the compiler plugin flags to ignore these Kotlin/Compose version checks, but that might bring some instability.